storage.py

```diff
@@ -56,3 +57,25 @@ async def save_object(
 		)
 		obj._id = result.inserted_id
 		log.debug(f"Inserted {type(obj)} ({obj._id})")
+
+
+async def get_pokemon(id: ObjectId) -> Pokemon:
+	# not sure if my type annotations are correct here
+	"""Get pokemon from the database by its ObjectId.
+
+	:param id: The id of the pokemon the user is trying to retrieve
+
+	:raises:
+		Exception: The pokemon with the supplied id cannot be found/does not exist
+
+	:returns: pokemon
+	"""
+	coll = pokemon()
+	# I'm not too confident I'm doign this correctly. Should I be doing these inside conditionals like in save_object?
+	result = await coll.find_one({'_id': ObjectId(id)})
```

Comment:
id is already an `ObjectId`, so isn't the cast unnecessary?

Reply:
Ah, genuinely didn't know, will fix that!